src/bundle/Core/Command/CopySubtreeCommand.php

```diff
@@ -168,20 +163,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     }
 
     /**
-     * @param \Ibexa\Contracts\Core\Repository\Values\Content\Location $location
-     *
-     * @return int
-     *
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
      */
-    protected function getAllChildrenCount(Location $location): int
+    protected function getAllChildrenCountExpr(Location $location): string
     {
         $query = new LocationQuery([
             'filter' => new Criterion\Subtree($location->pathString),
         ]);
 
-        $searchResults = $this->searchService->findLocations($query);
+        $totalCount = $this->searchService->findLocations($query)->totalCount;
 
-        return $searchResults->totalCount;
+        return $totalCount !== null ? (string) $totalCount : '∞';
```

Comment:
```suggestion
        return $totalCount !== null ? (string) $totalCount : '~';
```
or even `?` or `-` seems like a better choices then `∞`, but maybe either is ok.